src/modules/dashboard/components/ColonyHome/ColonyMembers/MembersSubsection.tsx

```diff
@@ -0,0 +1,233 @@
+import React, { useCallback, useMemo } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import Maybe from 'graphql/tsutils/Maybe';
+
+import NavLink from '~core/NavLink';
+import Heading from '~core/Heading';
+import { Tooltip } from '~core/Popover';
+import Icon from '~core/Icon';
+
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import useAvatarDisplayCounter from '~utils/hooks/useAvatarDisplayCounter';
+import { Colony, useLoggedInUser, BannedUser, UserProfile } from '~data/index';
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import { Address } from '~types/index';
+import { useTransformer } from '~utils/hooks';
+import { getAllUserRoles } from '~modules/transformers';
+import { hasRoot, canAdminister } from '~modules/users/checks';
+
+import styles from './ColonyMembers.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.title',
+    defaultMessage: `{isContributors, select,
+      true { Contributors }
+      other { Watchers }
+    } {hasCounter, select,
+        true { ({count})}
+        false {}
+      }`,
+  },
+  loadingData: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.loadingData',
+    defaultMessage: 'Loading members information...',
+  },
+  reputationFetchFailed: {
+    id: `dashboard.ColonyHome.ColonyMembers.MembersSubsection.reputationFetchFailed`,
+    defaultMessage: `Failed to fetch the colony's {isContributors, select,
+      true { contributors }
+      other { watchers }
+    }`,
+  },
+  tooltipText: {
+    id: 'dashboard.ColonyHome.ColonyMembers.MembersSubsection.tooltipText',
+    defaultMessage: `{isContributors, select,
+      true {Contributors are members of the Colony who have earned reputation.}
+      other { Watchers are members of the Colony
+         who currently don’t have reputation. }
+    }`,
+  },
+});
+
+type BannedMember = Maybe<
+  Pick<BannedUser, 'id' | 'eventId' | 'banned'> & {
+    profile?:
+      | Maybe<Pick<UserProfile, 'displayName' | 'username' | 'walletAddress'>>
+      | undefined;
+  }
+>;
```

Comment:
Maybe extract this somewhere externally, as I have a feeling we'll be reusing this.

Reply:
We won't, it was just temporary till I merge another PR where I move the data to the query. I've already deleted it in that PR.